packages/synapse-react-client/src/components/DraggableDialogue/DraggableDialogue.tsx

```diff
@@ -0,0 +1,38 @@
+import Draggable from 'react-draggable'
+import { Box, Paper } from '@mui/material'
+import { useRef } from 'react'
+import { SynapseChat } from '@/components/SynapseChat/index'
+
+export default function DraggableDialogue() {
```

Comment:
Rename for consistency with other components (e.g. MUI Dialog)
```suggestion
export default function DraggableDialog() {
```